src/Native/CMakeLists.txt

```diff
@@ -149,11 +155,20 @@ endif(CMAKE_SYSTEM_NAME STREQUAL SunOS)
 if(CLR_CMAKE_PLATFORM_UNIX)
     if(STRIP_SYMBOLS)
         find_program(STRIP strip)
-        if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
-            find_program(DSYMUTIL dsymutil)
-        else(CMAKE_SYSTEM_NAME STREQUAL Darwin)
-            find_program(OBJCOPY objcopy)
-        endif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+
+        if(CMAKE_SYSTEM_NAME STREQUAL Linux AND DEFINED CMAKE_TOOLCHAIN_FILE)
+            find_program(OBJCOPY
+                NAMES llvm-objcopy
+                llvm-objcopy${CLR_CMAKE_COMPILER_FILE_NAME_VERSION})
+            #find_program(OBJCOPY objcopy HINTS ${CROSS_ROOTFS}/usr/bin)
```

Comment:
Why is this commented out?

Reply:
Good catch. That was from my testing and is now removed, code not pushed yet.

Reply:
Code pushed.